scripts/build_test.py

```diff
@@ -78,9 +79,11 @@ class BuildTests(test_utils.GenericTestBase):
     """Test the build methods."""
 
     def tearDown(self) -> None:
-        super().tearDown()
         build.safe_delete_directory_tree(TEST_DIR)
         build.safe_delete_directory_tree(EMPTY_DIR)
+        pathlib.Path.unlink(pathlib.Path('mock_app.yaml'), missing_ok=True)
```

Comment:
This fixes https://github.com/oppia/oppia/discussions/21459#discussioncomment-11597123 which I also encountered while running tests for this PR.